@here/harp-examples/src/getting-started_free-camera.ts

```diff
@@ -280,7 +280,8 @@ Press 'V' to change the scene point of view<br>`;
         document.body.appendChild(message);
 
         const canvas = document.getElementById("mapCanvas") as HTMLCanvasElement;
-        const geoCenter = new GeoCoordinates(52.518611, 13.376111);
+        // Set the view over Geneva.
+        const geoCenter = new GeoCoordinates(46.207, 6.147, 2000);
```

Comment:
How is this fixing the issue? How is it related to the location? Is it b/c of the altitude?
Maybe you can also switch this example to use the look-at API

Reply:
instead of changing the example, can the far plane be fixed? It otherwise makes this example hard to maintain . (because if someone changes this position in future, or uses it as a base for another example, they may not be aware of the restriction of the position).